packages/components/src/local/react-table/index.tsx

```diff
@@ -150,11 +150,11 @@ export const ReactTable: React.FC<ReactTableProps> = (props) => {
    * @param appliedFilter: { col: string, filter: CellFilter[] }[]
    * @returns Row[]
    */
-  const getFilteredRows = (appliedFilters: { col: string, filter: CellFilter[] }[]): Row[] => {
-    const acceptableValues: string[] = flattenDeep(appliedFilters.map(filter => filter.filter.map(f => f.label)))
+  const getFilteredRows = (appliedFilter: { col: string, filter: CellFilter[] }[]): Row[] => {
+    const acceptableValues: { label: string, col: string }[] = flattenDeep(appliedFilter.map(filter => filter.filter.map(f => ({ label: f.label, col: filter.col }))))
     return rows.filter(row => {
       // see if this value is in the acceptable values
-      return Object.keys(row).some(key => acceptableValues.includes(row[key]))
+      return acceptableValues.some(({ label, col }) => label.includes(row[col]))
```

Comment:
I _may_ be mistaken, but...   If the `acceptable` value is `Closed` , and the row status is `Not Closed`, then `includes` means it will be a match won't it?

Please correct me if I'm wrong, but using `===` seems better because:
* only exact matches pass the test
* it _may_ be quicker, since it's just comparing the hash, rather than a character by character comparison.

Reply:
> value is Closed , and the row status is Not Closed

ahh, okay @IanMayo, in this case we should use `===`, I will updated it 👍